engine/faq.md

```diff
@@ -186,7 +186,7 @@ Because of this, and because setting `DEBIAN_FRONTEND` to `noninteractive` is
 mainly a 'cosmetic' change, we *discourage* changing it.
 
 If you *really* need to change its setting, make sure to change it back to its
-[default value](https://www.debian.org/releases/stable/i386/ch05s03.html.en){: target="_blank" class="_"}
+[default value](https://www.debian.org/releases/bullseye/s390x/ch05s02.en.html){: target="_blank" class="_"}
```

Comment:
Perhaps use `stable` instead? https://www.debian.org/releases/stable/amd64/ch05s02.en.html

Reply:
Probably good to link to the anchor of the section describing options; https://www.debian.org/releases/stable/amd64/ch05s03.en.html#installer-args

Reply:
> Probably good to link to the anchor of the section describing options; https://www.debian.org/releases/stable/amd64/ch05s03.en.html#installer-args

that's perfect